### ISSUE TITLE
Preact example is missing dependency "preact-render-to-string"

### ISSUE THREAD

**Issue by ole-treichel** [capri-js/capri#43]
When I create a preact project with `npm init capri my-capri-site -- -e preact`, the build fails because `preact-render-to-string` is not installed.

Edit: I just tested by cloning the capri repo and then the example works (after `npm install` in root). 

I am using `yarn`  instead of `npm`. Maybe that is the problem. I see that `@capri-js/preact` depends on `preact-iso`, which in turn depends on `preact-render-to-string`. 

Node: `16.13.2`
Yarn: `1.22.15`

**Comment by fgnass** [comment]
Good catch, thank you! I think I missed it, because as you wrote, it works when running things from within the checked-out repo.

As `preact-iso` lists `preact-render-to-string` only as peer dependency, I now added it as proper dependency to `@capri-js/preact`.

Another thing I noticed is, that `yarn build` currently fails in the preact example due to a bug in wouter. I opened a PR to fix this, too: https://github.com/molefrog/wouter/pull/272
